src/common/modals/InstanceExport/ExportPackModal.module.css

```diff
@@ -0,0 +1,34 @@
+.container {
+    height: 85%;
+    width: 100%;
+    /* font-family: 'GlacialIndifferenceRegular'; */
+    padding: 20px;
+    overflow-y: auto;
+  }
+  
+  .selectFolderBtn {
+    margin-top: 80px;
+    min-width: 130px;
+  }
+  
+  .continueBtn {
+    position: absolute;
+    bottom: 20px;
+    right: 20px;
+  }
+  
+  .backBtn {
+    position: absolute;
+    bottom: 20px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5447870)